app/assets/stylesheets/application.scss

```diff
@@ -114,50 +132,93 @@ p {
   -moz-transition: all 0.6s;
   transition: all 0.6s;
 }
-.btn-signup, .btn-signup:visited {
-  text-decoration: none;
-  color: white;
-  background-color: #cd2027;
-  font-size: 3em;
-  height: 100px;
-  width: 300px;
-  border: 2px solid white;
-}
-.btn-signup a {
-  color: white;
-  -webkit-transition: all 0.6s;
-  -moz-transition: all 0.6s;
-  transition: all 0.6s;
+#username {
+	text-align: center;
+	text-transform: capitalize;
+	color: #550000;
 }
-.btn-signup:hover {
-  border: 2px solid #cd2027;
-  background-color: white;
+#invite {
+	text-align: center;
+	color: white;
+	font-size: 3em;
 }
-.btn-signup:hover a{
-  text-decoration: none;
-  color: #cd2027;
+#total-amount {
+	text-align: left;
+	margin-left: 5em;
+	color: white;
+	font-family: "Arvo", serif;
+	font-size: 2em;
+	padding: 2em;
 }
 // Styling for the "SIGN UP" & SIGN IN forms
 .sign-in, .sign-in label, .sign-in input, .sign-in h2 {
   text-align: center;
   font-size: 125%;
   margin: 3em auto 0 auto;
-  text-transform: uppercase;
 }
 .sign-up, .sign-up label, .sign-up input, .sign-up h2 {
   text-align: center;
   font-size: 125%;
-  margin: 3em auto 0 auto;
-  text-transform: uppercase;
+  margin: 1em auto 0 auto;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `1em auto 0` instead of `1em auto 0 auto`